docs/DEVGUIDE.md

```diff
@@ -45,7 +45,7 @@ To clone the repository:
 First `cd` to the root of the cloned repository tree.
 To build the service-catalog you have two options:
 * `make build`
-* `DOCKER=1 make build`
+* `build/run.sh make build`
```

Comment:
@duglin I do like your idea of switching from `DOCKER=1` (to use it) to `NO_DOCKER=1` (to not use it) very, very much. 👍 

That being said... a huge component of this PR was to make us more consistent with how k8s addresses this same issue. `build/run.sh` is a page directly from their playbook.

That doesn't make me like your idea any less. I could go either way on this. Let's let @pmorie make the call one which way to go.